docs/content/releases.md

```diff
@@ -6,6 +6,52 @@ doctypes: ["concept"]
 toc: true
 docs: "DOCS-616"
 ---
+## NGINX Ingress Controller 2.4.0
+
+04 Oct 2022
+
+OVERVIEW:
+
+* Optimize reloads for the Ingress Controller by with [batch reloads at runtime](https://github.com/nginxinc/kubernetes-ingress/pull/2986).
+* Added support for [watching multiple namespaces](https://github.com/nginxinc/kubernetes-ingress/pull/2914). This can configured by passing a comma separated list of namespaces to the `-watch-namespace` CLI argument (e.g. `-watch-namespace=ns-1,ns-2`).
+* Display [year in logs](https://github.com/nginxinc/kubernetes-ingress/pull/2884) using the `-include-year` CLI argument. Example output: `I20220512 09:20:42.345457`.
+* TransportServer now supports the [proxy_protocol header on port 443 when TLS Passthrough is enabled](https://github.com/nginxinc/kubernetes-ingress/pull/2993).
+* TransportServer now supports [external name services](https://github.com/nginxinc/kubernetes-ingress/pull/3041), allowing users to configure an external resolver for upstreams that use a domain name. For examples, see [externalname-services](https://github.com/nginxinc/kubernetes-ingress/tree/release-2.4/examples/custom-resources/externalname-services).
+* Default server secret can now be [omitted from Helm based installs](https://github.com/nginxinc/kubernetes-ingress/pull/3087).
+* VirtualServer now supports [wildcard hostname](https://github.com/nginxinc/kubernetes-ingress/pull/2939).
+
+FEATURES:
+
+* [2831](https://github.com/nginxinc/kubernetes-ingress/pull/2831) Add ServiceMonitor to Helm Chart.
+* [2855](https://github.com/nginxinc/kubernetes-ingress/pull/2854) Add initialDelaySeconds to helm charts.
+* [2979](https://github.com/nginxinc/kubernetes-ingress/pull/2979) Allow to specify image with digest in helm chart.
+* [3031](https://github.com/nginxinc/kubernetes-ingress/pull/3031) Adding automountServiceAccountToken to helm chart.
```

Comment:
In previous releases, we have been putting Helm related changes in their own block (see line 118 below). I think we should keep the same pattern here and move these 3 enhancements to a Helm section.